docs/source/AdministratorGuide/Resources/identityprovider.rst

```diff
@@ -0,0 +1,59 @@
+.. _resourcesIdProvider:
+
+=================
+Identity Provider
+=================
+
+An identity provider (IdP) is a system entity that creates, maintains, and manages identity information for principals and also provides authentication services to relying applications within a federation or distributed network.
+IdP offer user authentication as a service. DIRAC has its own user base, but it is not an IdP as a service, therefore, it uses third-party IdP for authorization.
```

Comment:
I find this phrase rather confusing. It should be either clarified, or removed.

Reply:
removed